Controller/Index/index.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace SomethingDigital\AuctionItem\Controller\Index;
+
+use Composer\DependencyResolver\Request;
+use Magento\Framework\App\ActionInterface;
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\View\Result\PageFactory;
+/**
+ * Custom Controller for Auction Item display Page
+ *
+ * Class Index
+ * @package SomethingDigital\AuctionItem\Controller\Index
```

Comment:
This is the first time I'm seeing a package tag.

TIL: https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html#documentation-space

I guess it's not compatible with namespace organization so we should get rid of it